### DISCUSSION TITLE
Bonnes pratiques

### DISCUSSION THREAD

**Discussion by m-maillot** [SocialGouv/code-du-travail-numerique#3812] [Technique]
Je propose que l'on démarre une discussion sur les bonnes pratiques dans le code. 

L'idée est d'ajouter à la documentation du projet, les bonnes pratiques que l'on utilise dans le projet.

@carolineBda  @maxgfr Si vous avez des liens / références vers des bonnes pratiques déjà rédigé et qui peuvent nous être utile :)

**Comment by m-maillot** [comment]
Je commence avec le lien cheatsheet React + TS qui devrait un must read !

https://react-typescript-cheatsheet.netlify.app/

**Comment by maxgfr** [comment]
Super initiative @m-maillot 

Je vous propose ces liens pour ma part, ceux-ci parlent de nommage en typescript : 

- https://github.com/basarat/typescript-book/blob/master/docs/styleguide/styleguide.md
- https://github.com/Microsoft/TypeScript/wiki/Coding-guidelines
